rootfs/mkrootfs_tweak.sh

```diff
@@ -16,7 +16,7 @@ cat > "$root/etc/inittab" << "EOF"
 EOF
 chmod 644 "$root/etc/inittab"
 
-mkdir -m 755 -p "$root/etc/init.d" "$root/etc/rcS.d"
```

Comment:
May I ask what the rationale for this change is? Doesn't seem semantic-preserving to me.

Reply:
There is a warning saying that the attributes are only applied to the last path created.
Given that `$root/etc/` already exist, suppress that warning by removing `-p`